sdoc.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ["README.md"]
 
   s.add_runtime_dependency("rdoc", "~> 5.0")
+  s.add_runtime_dependency("actionview", "~> 4.2")
```

Comment:
I think it is better to keep to not make a Rails dependency dependent on Rails itself. If we could copy the methods that we need I think it would be better.